common-service-core/src/test/resources/test-wrong-format-messages.yml

```diff
@@ -0,0 +1,5 @@
+messages:
+    - key: com.ca.mfaas.test.noArguments
+      number: CSC0001
+      level: ERROR
+      text: "No arguments message"
```

Comment:
what's an "arguments message"?  a message with arguments? maybe just "no argument"? hard to with little to no context